server/client.go

```diff
@@ -2118,8 +2144,12 @@ func (c *client) addSubToRouteTargets(sub *subscription) {
 	}
 }
 
+func collectQueueName(queues *[][]byte, queue []byte) {
+	*queues = append(*queues, queue)
```

Comment:
one liner, so why a separate function? And this causes weird *[][]

Reply:
Shoot.. wanted to change that. I originally thought thatI would have to check for duplicate, etc.. but ended-up leaving it as simple append. So will do it inline indeed.

Reply:
done